README.md

```diff
@@ -34,7 +34,7 @@ instructions [here][create-key] to create a service account, generate a private
 key, and download a JSON file containing the private key. Place the JSON file on
 the machine that will be mounting the bucket.
 
-[console]: console.developers.google.com
+[console]: https://console.developers.google.com/project/_/storage/browser
```

Comment:
Thanks for fixing the broken link. This isn't the right bit of the console to point at though; the link is about generation credentials.